src/components/CommentAdder.jsx

```diff
@@ -1,23 +1,48 @@
 import { useState } from "react";
-
-const CommentAdder = () =>{
+import { useParams } from "react-router-dom";
+import { postComment } from "../utils/api";
+const CommentAdder = ({ setComments }) =>{
+    const { review_id } = useParams();
     const [newComment, setNewComment] = useState('');
+   
     const handleSubmit = (e) =>{
         e.preventDefault();
-        // postComment(newComment)
-        // .then
-    };
+    const newUser= {
+                username: "jessjelly",
+                body: e.target[0].value,
+            }
+            if (e.target[0].value.length===0) return alert("Empty text field, please try again.")
```

Comment:
You have a state of `newComment`. You should use this instead of going back to the DOM with `e.target[0].value`. Do this in all cases.